source/sessions/snapshot-sessions.rst

```diff
@@ -263,13 +263,13 @@ programs that don't need snapshot reads don't have to be changed.
 This goal is met by defining a ``SessionOptions`` field that applications use to
 start a ``ClientSession`` that can be used for snapshot reads. Alternative explicit approach of
 obtaining ``atClusterTime`` from ``cursor`` object and passing it to read concern object was considered initially.
-Session based approach was chosen as it aligns better with the existing API, and requires minimal API changes.
-Future extensibility for snapshot reads would be better served by session based approach, as no API changes will be required.
+A session-based approach was chosen as it aligns better with the existing API, and requires minimal API changes.
+Future extensibility for snapshot reads would be best served by a session-based approach, as no API changes will be required.
 
 Backwards Compatibility
 =======================
 
-The API changes to support sessions extend the existing API but do not
+The API changes to support snapshot reads extends the existing API but does not
```

Comment:
I assumed "to support sessions" was copypasta from the other session spec.

Reply:
copypasta indeed.
Should it stay "extend" and "do"?

Reply:
```suggestion
The API changes to support snapshot reads extend the existing API but do not
```